SAPHana/srHook/SAPHanaSrMultiTarget.py

```diff
@@ -27,14 +28,37 @@
     [trace]
     ha_dr_saphanasr = info
 """
-fhSRHookVersion = "0.181.0.0110.1727"
-srHookGen = "2.1"
+fhSRHookVersion = "0.180.0.1019.1151"
```

Comment:
wrong Hook version. Should be 0.0181. at least

Reply:
Yes already solved by new commit. New version will be 0.181.0.0216.1706.